File: 231117/벌금은 누구에게/who-will-pay.py
n,m,k=map(int,input().split())
student=[0]*(n+1)

flag=0
for _ in range(m):
    num=int(input())
    student[num]+=1
    if student[num]>=k:
        print(num)
        flag=1
        break

if flag==0:
    print(-1)

File: 251209/벌금은 누구에게/who-will-pay.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const [n,m,k] = input[0].split(' ').map(Number);
const students_num = input.slice(1).map(Number);

const arr = Array(n+1).fill(0);

let ans=-1;
for(let num of students_num){
    arr[num]++;
    if(arr[num]>=k){
        ans=num;
        break;
    }
}

console.log(ans)